src/frontend/components/SøknadsSteg/VelgBarn/LeggTilBarn/NyttBarnKort.test.tsx

```diff
@@ -56,15 +56,19 @@ test(`Kan legge til barn`, () => {
     const jaKnapp = getByText('felles.svaralternativ.ja');
     act(() => jaKnapp.click());
 
-    const navnLabel = getByText('hvilkebarn.leggtilbarn.barnets-navn');
+    const fornavnLabel = getByText('hvilkebarn.leggtilbarn.fornavn.spm');
+    const etternavnLabel = getByText('hvilkebarn.leggtilbarn.etternavn.spm');
     const idnrLabel = getByText('hvilkebarn.leggtilbarn.fnr.spm');
-    expect(navnLabel).toBeInTheDocument();
+    expect(fornavnLabel).toBeInTheDocument();
+    expect(etternavnLabel).toBeInTheDocument();
     expect(idnrLabel).toBeInTheDocument();
-    const navnInput = navnLabel.nextElementSibling || new Element();
+    const fornavnInput = fornavnLabel.nextElementSibling || new Element();
+    const etternavnInput = etternavnLabel.nextElementSibling || new Element();
     const idnrInput = idnrLabel.nextElementSibling || new Element();
 
     act(() => {
-        fireEvent.input(navnInput, { target: { value: 'Sirius Svaart' } });
+        fireEvent.input(fornavnInput, { target: { value: 'Sirius' } });
+        fireEvent.input(etternavnInput, { target: { value: 'Svaart' } });
         fireEvent.input(idnrInput, { target: { value: '031159123456' } });
```

Comment:
Siden dette nok er eneste gang jeg får sjansen til å nevne det: Legg merke til at dette er et gyldig fnr for Sirius Svaart :grin:

Reply:
haha, woow! kult da :D